src/App.tsx

```diff
@@ -15,35 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  BY_LENGTH = 'length',
+  ALPHABETICALLY = 'alphabetically',
+  DEFAULT = '',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [sortOrder, setSortOrder] = useState<boolean>(true);
+
+  function prepareGoods(goods: string[], sortBy: string, order = true) {
+    const preparedGoods = [...goods];
+
+    switch (sortBy) {
+      case SortType.BY_LENGTH: {
+        preparedGoods.sort((goodA, goodB) => goodA.length - goodB.length);
+        break;
+      }
+
+      case SortType.ALPHABETICALLY: {
+        preparedGoods.sort((goodA, goodB) => goodA.localeCompare(goodB));
+        break;
+      }
+
+      default:
+        break;
+    }
+
+    return order ? preparedGoods : preparedGoods.reverse();
+  }
```

Comment:
The function `prepareGoods` is not starting with a verb, which is not following the [CODE STYLE] convention in the checklist that function names should start with a verb. Consider renaming it to something like `sortGoods` or `getSortedGoods` to clearly indicate its purpose.